ApplicationCode/FileInterface/RifFractureModelGeologicalFrkExporter.cpp

```diff
@@ -90,24 +91,24 @@ bool RifFractureModelGeologicalFrkExporter::writeToFile( RimFractureModelPlot* p
     }
 
     std::map<QString, std::vector<double>> values;
-    values["dpthlyr"]        = plot->calculateTrueVerticalDepth();
-    values["strs"]           = plot->calculateStress();
-    values["strsg"]          = plot->calculateStressGradient();
-    values["elyr"]           = plot->calculateYoungsModulus();
-    values["poissonr"]       = plot->calculatePoissonsRatio();
-    values["tuflyr"]         = plot->calculateKIc();
-    values["clyrc"]          = plot->calculateFluidLossCoefficient();
-    values["clyrs"]          = plot->calculateSpurtLoss();
-    values["pembed"]         = plot->calculateProppandEmbedment();
-    values["zoneResPres"]    = plot->calculateReservoirPressure();
-    values["zoneWaterSat"]   = plot->calculateImmobileFluidSaturation();
-    values["zonePorosity"]   = plot->calculatePorosity();
-    values["zoneHorizPerm"]  = plot->calculateHorizontalPermeability();
-    values["zoneVertPerm"]   = plot->calculateVerticalPermeability();
-    values["zoneTemp"]       = plot->calculateTemperature();
-    values["zoneRelPerm"]    = plot->calculateRelativePermeabilityFactor();
-    values["zonePoroElas"]   = plot->calculatePoroElasticConstant();
-    values["zoneThermalExp"] = plot->calculateThermalExpansionCoefficient();
+    values["dpthlyr"]        = fractureModel->calculator()->calculateTrueVerticalDepth();
```

Comment:
Consider introducing a local variable for calculator to avoid long duplicated text